### PR TITLE
Add persian in new fork

### PR DESCRIPTION
**Pull request by mhdzli**
Sorry to open a new pull request.
Due to U.S. sanctions on Iran Microsoft restricted my GitHub  account and google closed access to all of their developers services  for any IP from Iran.  So I had some problem rebasing my old fork.
I will continue to do my best here and update this repository.

### CODE REVIEW

**Comment by menny** [comment]
@mhdzli just to be clear: this code will replace https://play.google.com/store/apps/details?id=com.anysoftkeyboard.languagepack.persian

**Comment by mhdzli** [comment]
@menny 
In the code you mentioned some basic and essential Persian character are missed e.g: semi-space (the difference between می شود and می‌شود) and uses. Some arabic character are used instead of persian e.g: ي is arabic and ی is persian and a few numbers are also arabic. And most importantly the key order is not standard.
I needed a good foss keyboard for my android and you have don a great job with AnySoftKeyboard. But I was disappointed when I installed the persian layout. So I rolled up my sleeves 😄
Persian is a right to left language and the in bottom rows that are provided in AnySoftKeyboard the question mark is for left to right languages. I want to change the bottom row question mark to fit Persian layout and also add semi-space when switch to persian layouts and I need you to help me with that if it's possible also for numbers and special characters layout eather. 
For now the bottom row in Persian layout is fixed and doesn't change with the setting option.

File: languages/persian/apk/build.gradle [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = false
```
Comment (menny) [inline languages/persian/apk/build.gradle]: This should be `true`, since we are going to use this code in production

Comment (mhdzli) [reply languages/persian/apk/build.gradle]: I've done the requested changes, if that's fine I resolve this conversation.

**Comment by menny** [comment]
Thank you! This is a great addition

**Comment by mhdzli** [comment]
You're welcome, thank you for the amazing job you've done here.

**Pull request merged**